ceph-nvmeof.conf

```diff
@@ -8,32 +8,29 @@
 #
 
 [gateway]
-
 name =
 group =
-addr = 127.0.0.1
+addr = ceph-nvmeof
```

Comment:
@epuertat 🖖

I am not sure about the motivation for  `addr = ceph-nvmeof` and [`hostname = ceph-nvmeof`](https://github.com/epuertat/ceph-nvmeof/blob/b4d55a95cfbaf4fb9d01950c2baba4594ab51276/docker-compose.yaml#L45).
- What are we trying to achieve?
- What are disadvantages of using `addr = 0.0.0.0` and using automatically assigned hostnames based on their service names?

Reply:
In this change I don't have yet your changes to the cli configuration, and I wanted to ensure that what I deliver here is fully functional. By using container hostnames we avoid the need of messing with IP addresses, and provide narrower exposure (`ceph-nvmeof` translates into 1 specific IP, which is valid both for server socket binding, and for client connection; but we don't bind to all interfaces, as with `0.0.0.0`). To be honest, this is not a big deal, since eventually Cephadm won't use any of this (this is only for development, testing and CI purposes).